.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request Message Format WG
+about: Suggest an idea or feature for Message Format
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe why does your solution should become a standard**
```

Comment:
nit: Instead of "become a standard", can we say "shape the standard"?  I don't know if any one pre-existing project or proposal is going to be a full and complete standard, which is what the wording of the question kind of sounds like.  Feel free to disagree or ignore.